marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/mappings.sjs

```diff
@@ -91,5 +102,6 @@ module.exports = {
     getPermissions,
     getArtifactNode,
     validateArtifact,
-    getArtifactSettingNode
+    getArtifactSettingNode,
+    defaultArtifactSettings
```

Comment:
Shouldn't there be a unit test against getArtifactSettings for a mapping to verify that the defaults are returned if there aren't settings yet?

Reply:
Added to the ML unit test to test the retrieval of default settings.